fetch.bs

```diff
@@ -3871,6 +3980,10 @@ optional <i>CORS-preflight flag</i>, run these steps:
 
        <li><var>request</var>'s <a for=request>redirect mode</a> is not "<code>follow</code>" and
        <var>response</var>'s <a for=response>URL list</a> has more than one item.
+
+       <li><p><var>request</var>'s <a for=request>response tainting</a> is "<code>opaque</code>" and
+       the <a>cross-origin resource policy check</a> with <var>request</var> and <var>response</var>
+       returns <b>blocked</b>.
```

Comment:
Instead of calling it twice both here and in the HTTP layer, could we call it in a step before "If actualResponse’s status is a redirect status, then:"? Then we would only have to check the response once I think, whether it came from a service worker or the network.

It seems that would miss the 401/407 scenarios, but I wonder if those should not also invoke HTTP fetch (maybe the latter shouldn't though) as they end up bypassing the service worker as currently defined. Hmm.

Either way we should document what we're doing and why, potentially just in a source comment for now.